hx_lti_assignment/templates/hx_lti_assignment/create_new_assignment2.html

```diff
@@ -21,76 +21,72 @@
 			<p>You'll need some source materials to create an annotation assignment!</p>
 		{% else %}
 			<div class='save' id='reorder-list-button'>Reorder list</div>
-			{% for source in targets_form %}
-				{% for choice in form.fields.assignment_objects.queryset %}
-					{% if source.target_object.value == choice.id %}
-						<div class='source-item' id='item-{{forloop.parentloop.counter0}}' data-order='{{source.order.value}}' data-id='{{source.target_object.value}}' data-aTarget="{{source.id.value}}" data-media="{{source.target_type.value}}">
-							<div class='first-row'>
-								<i class='fa fa-caret-right fa-lg' style='color:rgb(55, 124, 182);'></i> <span class='source-title'>{{choice.target_title}}</span>
-								<div class='order-div'><input type="number" class='ordernum' style='display:none;' min="1" value="{{source.order.value}}"><a href='#' style='display: none' class='order-change-button'>Move</a></div>
+			{% for source in form.instance.assignmenttargets_set.all %}
```

Comment:
@lduarte1991 I just pushed those changes. 

In addition to those changes, I discovered that `targets_form` was adding 3 extra form widgets beyond whatever `AssignmentTargets` were supposed to be in the formset. This caused an error when the template tried to access `source.instance.target_object.pk` (no model instance attached), so to I had to set `extra=0` for the `AssignmentTargetsFormSet`. One other issue that came up was that when an existing assignment was not valid, the template complained because it was missing the `assignment_id` parameter, so I added that to the `render()` context.